### PR TITLE
making a change to the api to provide all available links if no paw i…

### PR DESCRIPTION
**Pull request by brianedmonds90**
…s provided

### CODE REVIEW

**Comment by privateducky** [comment]
why?

File: app/service/rest_svc.py [hunk 1]
```diff
@@ -162,11 +162,14 @@ def __init__(self):
...
                    if p.is_file() and not p.name.startswith('.'))
 
-    async def get_potential_links(self, op_id, paw):
+    async def get_potential_links(self, op_id, paw=None):
         operation = (await self.get_service('data_svc').locate('operations', match=dict(id=op_id)))[0]
         if operation.finish:
             return []
-        agents = await self.get_service('data_svc').locate('agents', match=dict(paw=paw))
+        if paw:
+            agents = await self.get_service('data_svc').locate('agents', match=dict(paw=paw))
```
Comment (privateducky) [inline app/service/rest_svc.py]: you can do this in 1 line

**Comment by brianedmonds90** [comment]
The reasoning behind this PR is to allow for the 'manual' planner to have one endpoint that it can hit to deliver all potential links that are available at a given point in time.

File: app/service/rest_svc.py (hunk 1 above)
Comment (brianedmonds90) [reply app/service/rest_svc.py]: Good call. Feedback is now addressed

**Pull request merged**